src/common.js

```diff
@@ -82,8 +82,10 @@ export const useCheckTearing = () => {
   });
 };
 
-export const createApp = (useCount, useIncrement, useDouble, Root = React.Fragment) => {
-  const Counter = React.memo(() => {
+export const createApp = (useCount, useIncrement, useDouble, Root = React.Fragment,
+  doUseTransition = useTransition, componentWrapper = React.memo) => {
```

Comment:
custom wrapper might be be acceptable, but custom useTransition doesn't feel right. 🤔